docs/internals/optimizer.rst

```diff
@@ -1096,6 +1097,48 @@ The step LiteralRematerialiser is not required for correctness. It helps deal wi
 ``function f(x) -> y { revert(y, y} }`` where the literal ``y`` will be replaced by its value ``0``,
 allowing us to rewrite the function.
 
+.. _unused-store-eliminator:
+
+UnusedStoreEliminator
```

Comment:
I'd add an index entry for it. We should really have index entries for all of them but I guess that's out of scope of this PR.

Reply:
How and where do I do that?

Reply:
Check on other pages how the `index` directive is used. It's often next to headers.

Also see the "Keyword Index" section in the docs menu to see how the index looks like.